src/platform-integration/windows/extern_win.md

```diff
@@ -0,0 +1,21 @@
+---
+title: External windows in Flutter Windows apps
+description: Special considerations for adding external windows to Flutter apps
+---
+
+## Windows lifecycle
+

```

Comment:
I'd be tempted to break out sections:

## Who is affected
Windows applications built against Fluter version X or later that open non-Flutter windows.